ingestion_tools/scripts/importers/annotation.py

```diff
@@ -282,25 +305,37 @@ def convert(self, output_prefix: str):
             write_mrc=self.config.write_mrc,
             write_zarr=self.config.write_zarr,
             voxel_spacing=self.get_voxel_spacing().as_float(),
+            scale_0_dims=output_dims,
+            threshold=self.threshold,
         )
 
 
 class SemanticSegmentationMaskAnnotation(VolumeAnnotationSource):
     shape = "SegmentationMask"  # Don't expose SemanticSegmentationMask to the public portal.
     mask_label: int
+    rescale: bool = False
+    threshold: float | None
 
     def __init__(
         self,
         mask_label: int | None = None,
+        rescale: bool = False,
+        threshold: float | None = None,
```

Comment:
Not a blocker: Based on the offline discussion, one opportunity for refactor would be to create a new annotation type for cases where the source is a probability map.